server.js

```diff
@@ -45,6 +45,13 @@ app.prepare().then(() => {
             : pathname)
       })
       res.end()
+    } else if (req.headers.host === 'err.dvc.org') {
+      // err.dvc.org/{hdr} -> dvc.org/doc/user-guide/troubleshooting#{hdr},
+      res.writeHead(301, {
```

Comment:
302 is better here

Reply:
I'm so confused by redirect codes at this point haha. I thought we decided having 301 everywhere was best. See https://github.com/iterative/dvc.org/issues/768#issuecomment-549486994

I have nothing agains 302 here but what about consistency vs. the`man.` redirect and others?

Sorry, I know it's a completely separate discussion to the troubleshooting guide but... Just saying.

Reply:
ok, so in the discussion and snippet there are two things happening simultaneously. First is a backward compatible URLs like `man.dvc.org/remote-add` - and I was talking about them mostly - we don't care if they are 301 or 302 - we can even completely disable those now. Second is the actual layer of indirection for URLs like `man.dvc.org/remote/add` for those (ideally) we want to preserve the ability to being able change the location they redirect two. Considering that 301 are so aggressively cached (I didn't know about that - thus we probably had 301s everywhere), we should pick a code for `man.dvc.org` and `error.dvc.org` that is more appropriate. Like we discussed 302 is not ideal, either - so, it can be 303 or something else?

Reply:
> (we can even completely disable those now)

(Off-topic) We only have it for `/get-url` and `/import-url`. - Will remove that exception now. See 44fe4263

Reply:
> indirection for URLs like man.dvc.org/remote/add for those (ideally) we want to preserve the ability to change the location they redirect two. Considering that 301 are so aggressively cached, we should pick a code for man.dvc.org and error.dvc.org that is more appropriate?

302 Found [has been superseded](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#3xx_Redirection) by 303 See Other, and 307 Temporary Redirect. This is a conversation pertaining more to the HTTP redirect issue actually (https://github.com/iterative/dvc.org/issues/757#issuecomment-549494538). But anyway, let's decide?

I vote for [303 See Other](https://en.wikipedia.org/wiki/HTTP_303).

Reply:
k, let's try 303

Reply:
OK, done for existing ones in 29f3b686 of #915, and in 6630b464 for this PR.